cs-rin-ru-enhanced-mod.user.js

```diff
@@ -423,6 +438,7 @@ if (options.infinite_scrolling && $("[title='Click to jump to page…']").length
 function functionsCalledByInfiniteScrolls(data) {
```

Comment:
new previews setup missing (so now the previews do not show on pages loaded by infinite scrolling)